EIPS/eip-4800.md

```diff
@@ -0,0 +1,241 @@
+---
+eip: 4799
```

Comment:
Please rename the file to `EIPS/eip-4800.md`

Reply:
assuming this was a typo - file renamed to EIPS/eip-4799.md